src/AsmResolver.DotNet/Cloning/IMemberClonerListener.cs

```diff
@@ -0,0 +1,15 @@
+namespace AsmResolver.DotNet.Cloning
+{
+    /// <summary>
+    /// <see cref="MemberCloner"/> Callback Listener.
```

Comment:
Similar as above, we need to explain what the purpose is of this interface. i.e., that this interface's members receive notifications from the cloner.